src/styles/main.scss

```diff
@@ -1 +1,71 @@
 // put code here
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
```

Comment:
Create a separate file for stopwatch BEM block
![image](https://github.com/mate-academy/layout_stop-watch/assets/96409563/bb949b03-8c3c-4452-8018-ed0e212f67e3)